api/src/main/java/org/pmiops/workbench/cdr/dao/CBCriteriaDao.java

```diff
@@ -289,6 +289,7 @@ Long findDomainCountOnCodeAndStandard(
               + "where code like upper(concat(:term,'%')) "
               + "and is_standard = :standard "
               + "and domain_id in (:domains) "
+              + "and full_text like '%_rank1]' "
```

Comment:
No guarantee this is always at the very end.
```suggestion
              + "and full_text like '%_rank1]%' "
```

Reply:
They always end in '_rank1]'